01_Introduction/intro.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>Note that this is just making an example array</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='127'/>